src/containers/SearchContainer.jsx

```diff
@@ -40,18 +40,16 @@ export default class SearchContainer extends React.Component {
   onSearchClick = () => {
     const { searchMovies, setQuery } = this.props;
     const { searchQuery } = this.state;
-    if (searchQuery) {
-      setQuery({ searchQuery });
-      searchMovies(
-        { searchQuery, isSearch: true, isSearchChange: true },
-      );
-    }
+    setQuery({ searchQuery });
+    searchMovies(
+      { searchQuery },
+    );
   };
 
   onEnterPress = (e) => {
     const { setQuery } = this.props;
     const { searchQuery } = this.state;
-    if (e.charCode === ENTER_KEY && searchQuery) {
+    if (e.charCode === ENTER_KEY) {
       setQuery({ searchQuery });
```

Comment:
remove `setQuery`  from here please. You already call it in `onSearchClick`  method